packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -1,14 +1,17 @@
-const aws = require('./aws.js');
-const S3 = require('./aws-s3');
 const fs = require('fs');
 const path = require('path');
 const _ = require('lodash');
-const providerName = require('../../lib/constants').ProviderName;
+const BottleNeck = require('bottleneck');
+const chalk = require('chalk');
 const columnify = require('columnify');
+
+const aws = require('./aws.js');
+const S3 = require('./aws-s3');
+const providerName = require('../../lib/constants').ProviderName;
 const { formUserAgentParam } = require('./user-agent');
 
-const CFN_MAX_CONCURRENT_REQUEST = 15;
-const CFN_POLL_TIME = 7000; // 7sec
+const CFN_MAX_CONCURRENT_REQUEST = 5;
```

Comment:
Are we sure we want to change this to 5 from 15?

Reply:
We are using bottleneck and it can support 5 concurrent requests as there is at least 100ms gap between these requests. Also I have reduced the poll time to 5 seconds instead of 7 seconds and we don't have to wait 5 seconds for queue to fire next round of requests. POLL TIME is now used only to read root stack and not the ones that are in the queue